xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -0,0 +1,1828 @@
+   </step>
+ </procedure>
+ <note>
+   <para>
+     There will be no loss of functionality as regards the &o_objstore;
+     solution after the upgrade. The difference is that some additional
+     XFS features will not be available on file systems which were formatted
+     under SLE 12 SP3 or earlier. These XFS features aid in the detection of,
+     and recovery from, data corruption. They are enabled by default for XFS
+     file systems formatted under SLE 12 SP 4.
+   </para>
+ </note>
+ <procedure>
+   <title>Additional Pre-Upgrade Checks That Should Be Performed</title>
+   <para>
+     In addition to the automated upgrade checks above, there are some checks
+     that should be performed manually.
+   </para>
+   <step>
+     <para>
+       For each network interface device specified in the input model under
+       <filename>~ardana/openstack/my_cloud/definition</filename>, ensure that
+       there is only one untagged VLAN. The &productname; &productnumber; &clm;
+       configuration processor will fail with an error if it detects this problem
+       during the upgrade, so address this problem before starting the upgrade process.
+     </para>
+   </step>
+   <step>
+     <para>
+       If the deployer node is not a standalone system, but
+       is instead co-located with the DB services, this can lead to
+       potentially longer service disruptions during the upgrade process. To
+       determine if this is the case, check if the deployer node (OPS-LM--first-member)
+       is a member of the database nodes (FND-MDB). You can do this with the
+       following command:
+     </para>
+     <screen>&prompt.ardana;cd scratch/ansible/next/ardana/ansible/
+&prompt.ardana;ansible -i hosts/verb_hosts 'FND-MDB:&amp;OPS-LM--first-member' --list-hosts
+     </screen>
+     <para>
+       If the output is:
+     </para>
+     <screen>
+       No hosts matched
+     </screen>
+     <para>
+       Then the deployer node is not co-located with the database nodes.
+       Otherwise, if the command reports a hostname, then there may be
+       additional interruptions to the database services during the upgrade.
+     </para>
+   </step>
+   <step>
+     <para>
+       Similarly, if the deployer is co-located with the database services,
+       and you are also trying to run a local SMT service on the deployer
+       node, you will run into issues trying to configure the SMT to enable and
+       mirror the SLE 12 SP4 and &productname; &productnumber; repositories.
+     </para>
+     <para>
+       In such cases, it is recommended that you run the SMT services on a
+       different node, and NFS-import the <filename>/srv/www/htdocs/repo</filename> onto the deployer
+       node, instead of trying to run the SMT services locally.
+     </para>
+   </step>
+ </procedure>
+ <note>
+   <title>Backup the &clm; Configuration Settings</title>
+     <para>
+       The integrated backup solution in &productname; 8 &clm;, freezer, is no
```

Comment:
```suggestion
       The integrated backup solution in &productname; 8 &clm;, <systemitem class="service">freezer</systemitem>, is no
```

Reply:
@lproven Freezer is a service of OpenStack, not a systemitem. That's why we added the entity. I will update your comment to reflect that.